.github/workflows/cd.yml

```diff
@@ -874,7 +874,7 @@ jobs:
       }}
     runs-on: ubuntu-latest
     container:
-      image: grafana/docs-base:latest # zizmor: ignore[unpinned-images]
+      image: grafana/docs-base:latest@sha256:6fb6a6b1089885129fabef0728c5e96ed4de81ad60522a2045b426daaaa47068 # zizmor: ignore[unpinned-images]
```

Comment:
looks like this doesn't work properly? https://github.com/grafana/plugin-ci-workflows/blob/017fa07ea57b0bcdd0c0dc33bc3c5da6ae300d82/renovate.json#L11